### DISCUSSION TITLE
12-07

### DISCUSSION THREAD

**Discussion by miniposi** [ChristmasTest/christmas#52] [Daily_Meeting]
### 🎅🏻 회의 내용

**어제 다들 모하셨나요?**
- 수영님 : 강의 들었다.. 뭐했지... MongoDB 보고 운동 갔다 헬스인
- 수님 : 학교 프로젝트 마무리 / PT도 받았다 / 강의 보는 시도 했다
- 슬희님 : 센트리 짜증남;; / 디코방 하나 만들자
- 윤혁님 : 과제를 다 끝냄

### ❄️ 오늘 할일
* 강의 듣기
* 각자 맡은 일하기

### ⛄️ TMI

사실 전 이번주가 기말고사 기간이랍니다..😮‍💨

**Comment by hanseulhee** [comment]
헐 ~~~ 대학생 힘들겟다 🥺

**Reply by oauch** [reply]
대학생 아니세요..?
